src/meds/schema.py

```diff
@@ -29,73 +30,75 @@
 birth_code = "MEDS_BIRTH"
 death_code = "MEDS_DEATH"
 
-subject_id_field = "subject_id"
-time_field = "time"
-code_field = "code"
-numeric_value_field = "numeric_value"
 
-subject_id_dtype = pa.int64()
+class Data(PyArrowSchema):
```

Comment:
`MedsDataSchema`?

Reply:
What about just `DataSchema`? the `MEDS` part is captured inherently in the import structure. I don't feel strongly either way, though, to be honest, between any of the three options (`Data`, `DataSchema`, and `MEDSDataSchema`). @EthanSteinberg @prockenschaub @rvandewater @kamilest and others, what are your thoughts on this and the other renaming notes?

Reply:
Thanks! Yeah, DataSchema is also good (for me, a definite improvement on just Data)